service/models.py

```diff
@@ -15,69 +17,77 @@
 # Function to initialize the database
 def init_db(app):
     """ Initializes the SQLAlchemy app """
-    YourResourceModel.init_db(app)
+    ShopCarts.init_db(app)
 
 
 class DataValidationError(Exception):
     """ Used for an data validation errors when deserializing """
 
 
-class YourResourceModel(db.Model):
+class ShopCarts(db.Model):
     """
-    Class that represents a YourResourceModel
+    Class that represents a ShopCarts
     """
-
     app = None
-
     # Table Schema
-    id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
+    id = db.Column(db.Integer, Identity(start=1, cycle=True), primary_key=True)
+    # Should add db.ForeignKey(customer.id) when integrate with Customer.
+    customer_id = db.Column(db.Integer, index=True)
+    # Should add db.ForeignKey(product.id) when integrate with product.
+    product_id = db.Column(db.Integer)
+    quantities = db.Column(db.Integer)
+    # price = db.Column(db.Numeric(10, 2))
 
     def __repr__(self):
-        return f"<YourResourceModel {self.name} id=[{self.id}]>"
+        return f"<ShopCarts customer_id=[{self.customer_id} product_id=[{self.product_id}]>"
 
     def create(self):
         """
-        Creates a YourResourceModel to the database
+        Creates a ShopCarts record to the database.
         """
-        logger.info("Creating %s", self.name)
-        self.id = None  # pylint: disable=invalid-name
+        logger.info("Creating a ShopCarts record for customer %d with product %d quantities %d",
+                    self.customer_id, self.product_id, self.quantities)
+        # self.id = None  # pylint: disable=invalid-name
         db.session.add(self)
         db.session.commit()
 
     def update(self):
         """
-        Updates a YourResourceModel to the database
+        Updates a ShopCarts record to the database
         """
-        logger.info("Saving %s", self.name)
+        logger.info("Saving a ShopCarts record for customer %d with product %d quantities %d",
+                    self.customer_id, self.product_id, self.quantities)
```

Comment:
An error should be thrown here if the shopcart record doesn't exist.

Reply:
You are right. I should add it.